lib/FileUploader/FileUploader.js

```diff
@@ -73,6 +73,7 @@ export const FileUploader = ({
                   marginBottom0
                   buttonStyle="primary"
                   onClick={open}
+                  autoFocus
```

Comment:
https://github.com/folio-org/ui-data-export/blob/master/CODESTYLEGUIDE.md#ordering-of-destructuring-parameters

The same rule applies to the order of props. Please adjust.